src/Workspaces/Remote/Core/Diagnostics/IPerformanceTrackerService.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Host;
+
+namespace Microsoft.CodeAnalysis.Remote.Diagnostics
+{
+    internal interface IPerformanceTrackerService : IWorkspaceService
+    {
+        void AddSnapshot(IEnumerable<AnalyzerPerformanceInfo> snapshot, int unitCount);
+        void GenerateReport(List<ExpensiveAnalyzerInfo> badAnalyzers);
+
+        event EventHandler SnapshotAdded;
+    }
+
+    internal struct ExpensiveAnalyzerInfo
+    {
+        public readonly bool BuiltIn;
+        public readonly string AnalyzerId;
+        public readonly string AnalyzerIdHash;
+        public readonly double LOF;
```

Comment:
❕ I do not know what `LOF` stands for. I am assuming it's an initialism; please find out what it stands for and expand the term 'lof' to PascalCase or camelCase as appropriate throughout the pull request.

Reply:
I am going to just change here. I don't see any value changing it for every place it shows up.  where LOF  is calculated, it has link to LOF wiki.